templates/patient/partials/portal.html.twig

```diff
@@ -13,6 +13,9 @@ The Patient Portal card for the Medical Record Dashboard
     {% if portalAuthorized == false %}
         {{ "Patient Portal Not Authorized"|xlt }}
     {% else %}
+        {% for row in content %}
+            {{ include(row.template, row.vars) }}
```

Comment:
so if someone wants to include content from within their module, they would need to add an additional path / file loader to the twig environment right?

Reply:
Correct, they would need to add whatever path they are using into the loader environment. Which, since the loader is going to get crowded, perhaps we should make it best practice to namespace all templates? That would help easy any future change in filesystem paths (https://twig.symfony.com/doc/3.x/api.html#built-in-loaders).